Corona-Warn-App/src/test/java/de/rki/coronawarnapp/appconfig/AppConfigProviderTest.kt

```diff
@@ -185,6 +185,16 @@ class AppConfigServerTest : BaseIOTest() {
         }
     }
 
+    @Test
+    fun `if supportedCountryList is empty, we insert DE as fallback`() {
+        coEvery { api.getApplicationConfiguration("DE") } returns APPCONFIG_BUNDLE.toResponseBody()
+        every { verificationKeys.hasInvalidSignature(any(), any()) } returns false
+
+        runBlocking {
+            createDownloadServer().getAppConfig().supportedCountriesList shouldBe listOf("DE")
+        }
+    }
+
```

Comment:
@d4rken this fallback also applies then when the list of countries is displayed on the UI, isn't it? I think for the UI, there's a different kind of fallback. The `DE` fallback is only for the submission and the key download (i.e. non-UI coding).

Reply:
I'll correct this. You are right, my approach would interfere with that behavior.